### PR TITLE
Add blog post for February standard community call

### LINKED ISSUE #152: February Standard for Public Code Community Call
Summarize notes to a blog post

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #152

(Boris and Eric reviewed the summary in the notes HackMD.)

-----
[View rendered _posts/2021-02-11-notes-from-community-call-4-february-2021.md](https://github.com/publiccodenet/blog/blob/ja-february-standard-blog/_posts/2021-02-11-notes-from-community-call-4-february-2021.md)

### CODE REVIEW

File: _posts/2021-02-11-notes-from-community-call-4-february-2021.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+* [Jan Ainali](https://publiccode.net/team/jan-ainali.html)
+* [Eric Herman](https://publiccode.net/team/eric-herman.html)
+* [Elena Findley-de Regt](https://publiccode.net/team/elena-findley-de-regt.html)
+* [Boris van Hoytema](https://publiccode.net/team/boris-van-hoytema.html)
+
+## Updates from the Foundation
+
+* Signalen is incubation for the Standard for Pubic Code
+* Signalen's and OpenZaak's assessments are both updated to the v.0.2.1 of the template
```
Comment (ElenaFdR) [inline _posts/2021-02-11-notes-from-community-call-4-february-2021.md]: please hyperlink [v.0.2.1 of the template]

File: _posts/2021-02-11-notes-from-community-call-4-february-2021.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## Proposed for this call
+
+* [How should we certify compliance?](https://github.com/publiccodenet/standard/issues/434)
+
+## Notes
+
+This community call was full of sharing visions of the future with a lot of ideas and bouncing them back and forth.
+Here's a summary of how we currently believe certification will look like for codebases that comply with the Standard for Public Code.
+
+The Foundation for Public Code will store information about which versions of all codebases that are compliant with the Standard.
```
Comment (ElenaFdR) [inline _posts/2021-02-11-notes-from-community-call-4-february-2021.md:41]: This sentence isn't quite right re English. Is this ok?
```suggestion
The Foundation for Public Code will store information organized by version for all codebases that are (or have ever been) compliant with the Standard.
```

**Review by ElenaFdR — CHANGES_REQUESTED** [review]
Nice summary of the (highly unstructured) call! I've done a round of stylistic editing/proofreading changes, and have 2 substantive changes (see comments). Happy to approve as soon as the comments are resolved!

File: _posts/2021-02-11-notes-from-community-call-4-february-2021.md (hunk 1 above)
Comment (Ainali) [reply _posts/2021-02-11-notes-from-community-call-4-february-2021.md]: Do we want to link to the empty template on our site, or each of the assessments?

File: _posts/2021-02-11-notes-from-community-call-4-february-2021.md (hunk 2 above)
Comment (Ainali) [reply _posts/2021-02-11-notes-from-community-call-4-february-2021.md:41]: Hmm, I am not trying to state how they are organized, but that what we will store is references to which versions are compliant. They will probably be organized chronologically (that is, they will be added in the order they are created).

Comment (ElenaFdR) [reply _posts/2021-02-11-notes-from-community-call-4-february-2021.md:41]: What about: "The Foundation for Public Code will store information for every version of all codebases that are (or have ever been) compliant with the Standard."

File: _posts/2021-02-11-notes-from-community-call-4-february-2021.md (hunk 1 above)
Comment (ElenaFdR) [reply _posts/2021-02-11-notes-from-community-call-4-february-2021.md]: Empty template!

**Review by ElenaFdR — APPROVED** [review]
Very nice!

**Pull request merged**